books/RayTracingInOneWeekend.html

```diff
@@ -2728,16 +2729,19 @@
 The reason we defocus blur in real cameras is because they need a big hole (rather than just a
 pinhole) to gather light. This would defocus everything, but if we stick a lens in the hole, there
 will be a certain distance where everything is in focus. You can think of a lens this way: all light
-rays coming _from_ a specific point at the focal distance -- and that hit the lens -- will be bent
+rays coming _from_ a specific point at the focus distance -- and that hit the lens -- will be bent
 back _to_ a single point on the image sensor.
 
-In a physical camera, the distance to that plane where things are in focus is controlled by the
-distance between the lens and the film/sensor. That is why you see the lens move relative to the
-camera when you change what is in focus (that may happen in your phone camera too, but the sensor
-moves). The “aperture” is a hole to control how big the lens is effectively. For a real camera, if
-you need more light you make the aperture bigger, and will get more defocus blur. For our virtual
-camera, we can have a perfect sensor and never need more light, so we only have an aperture when we
-want defocus blur.
+We call the distance between the projection point and the plane where everything is in perfect focus
+the _focus distance_. Be aware that the focus distance is not the same as the focal length -- the
+_focal length_ is the distance between the projection point and the image plane.
```

Comment:
You use projection plane instead of image plane. And then later in the book, it's written as:

```
We don’t need to simulate any of the inside of the camera. For the purposes of rendering an image outside the camera, that would be unnecessary complexity. Instead, I usually start rays from the surface of the lens, and send them toward a virtual film plane, by finding the projection of the film on the plane that is in focus (at the distance focus_dist). 
```

"virtual film plane"

It doesn't need to be rigorously consistent within the book. And, what we're referring to here (image plane/film plane/projection plane) has all three terms used interchangeably in the literature.
I'm fine with it as you've written, since I think it's still reasonably clear.

Reply:
Hmmm, excellent point. I dislike "film plane", and would prefer either "projection plane" or "image plane", but lean toward the latter. Reviewing...